src/discovery/discovery.py

```diff
@@ -19,7 +20,17 @@
   'https': '',
 }
 
+def load_city_locations():
+    """Load city location data from JSON file."""
+    with open("basestation/routers/satellites.json", "r") as file:
```

Comment:
Instead of opening this non existent file, you should use a list with 10-15 random city names.